lib/inferno/apps/web/controllers/test_sessions/create.rb

```diff
@@ -10,7 +10,9 @@ class Create < Controller
 
           def handle(req, res)
             params = req.params.to_h
-            params.merge!(JSON.parse(req.body.string).symbolize_keys) unless req.body.string.blank?
+            unless req.body.string.blank? || req.env['CONTENT_TYPE'].include?('multipart/form-data')
```

Comment:
Compound conditions with `unless` are really confusing, so you should change this to use `if`.